apps/journeys-admin/src/components/JourneyVisitorsList/JourneyVisitorsList.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactElement } from 'react'
+import Image from 'next/image'
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import LoadingButton from '@mui/lab/LoadingButton'
+import Typography from '@mui/material/Typography'
+import { useTranslation } from 'react-i18next'
+import { GetJourneyVisitors_visitors_edges as VisitorEdge } from '../../../__generated__/GetJourneyVisitors'
+import VisitorsPlaceholder from '../../../public/VisitorsPlaceholder.svg'
+import { VisitorCard } from './VisitorCard'
+
+interface Props {
+  visitorEdges?: VisitorEdge[]
+  visitorsCount?: number
+  fetchNext: () => void
+  loading: boolean
+  hasNextPage?: boolean
+}
+
+export function JourneyVisitorsList({
+  visitorEdges,
+  visitorsCount,
+  fetchNext,
+  loading,
+  hasNextPage = false
+}: Props): ReactElement {
+  const { t } = useTranslation('apps-journeys-admin')
+  const hasVisitors = visitorEdges != null && visitorEdges.length > 0
+  return (
+    <Stack
+      spacing={6}
+      alignItems="center"
+      sx={{ width: '100%', height: '100%' }}
```

Comment:
Add this to remove the margins for mobile
```suggestion
      sx={{ width: '100%', height: '100%' , mx: { xs: -6, sm: 0 }}}
```